src/ui/pages/search/SearchHeader.js

```diff
@@ -49,44 +71,74 @@ export default class SearchHeader extends React.Component {
-                    <div style={{width: 10}}></div>
-                    <div id="searchBoxLabelGreeting" className={this.props.searchBoxUp ? "searchBoxLabelGreetingSearch" : "searchBoxLabelGreetingHome" }>
-                        <span className="FontLatoBlack Fleft Fs30 DispBlock" style={{color: "#02a2f2"}}>Welcome to EAM Light</span>
-                    </div>
-                </div>
-                <div id="searchBoxInput" className={this.props.searchBoxUp ? "searchBoxInputSearch" : "searchBoxInputHome" }>
-                    <EAMBarcodeInput updateProperty={val => this.props.fetchDataHandler(val, entityTypes)} top={3} right={-7}>
-                        <input
-                            onInput={this.handleSearchInput.bind(this)}
-                            id="searchInputText"
-                            onKeyDown={this.props.onKeyDown}
-                            value={this.props.keyword}
-                            style={{textTransform: "uppercase"}}
-                            ref={(input) => { this.searchInput = input; }} />
-                    </EAMBarcodeInput>
-                    <FontIcon style={searchIconStyle} className="fa fa-search"/>
-                    {showTypes && <div style={{height: '30px', display: 'flex', direction: 'row', whiteSpace: 'nowrap'}}>
-                       {Object.values(SEARCH_TYPES).map(this.renderTypeCheckbox.bind(this))}
-                    </div>}
-                    <label id="searchPlaceHolder">{!this.props.keyword && "Search for Equipment, Work Orders, Parts, ..."}</label>
-                </div>
+            <div
+                id="searchBox"
+                className={this.props.searchBoxUp ? "searchBox searchBoxSearch" : "searchBox searchBoxHome"}
+                ref={searchBoxDiv => this.searchBoxDiv = searchBoxDiv}
+                style={this.props.searchBoxUp && this.state.isPhoneScreen ? { height: 'fit-content', paddingTop: '5px', paddingBottom: '5px' } : {}}
+            >
+                {
+                    this.props.searchBoxUp && this.state.isPhoneScreen ?
+                    <div id="searchBoxInput" className="searchBoxInputSearch" style={{ width: '100%'}}>
+                        {this.renderInput()}
+                        <div id="searchBoxLabel" className="searchBoxLabelHome" style={{ justifyContent: 'left'}}>
+                            {this.renderIcon()}
+                            {this.renderFilters()}
+                        </div>
+                    </div> :
+                    <>
+                        <div id="searchBoxLabel" className="searchBoxLabelHome">
+                            {this.renderIcon()}
+                        </div>
+                        <div id="searchBoxInput" className={this.props.searchBoxUp ? "searchBoxInputSearch" : "searchBoxInputHome" }>
+                            {this.renderInput()}
+                            {this.renderFilters()}
+                        </div>
+                    </>
```

Comment:
If possible, can this code be simplified so that we don't call the three render functions (renderIcon, renderInput, renderFilters) in two places?

Reply:
as we talked, it would break searchBox positioning